crates/vp_prelude/src/lib.rs

```diff
@@ -119,11 +111,10 @@ pub fn verify_signatures(ctx: &Ctx, tx: &Tx, owner: &Address) -> VpResult {
 }
 
 /// Checks whether a transaction is valid, which happens in two cases:
-/// - tx is whitelisted, or
-/// - tx is executed by an approved governance proposal (no need to be
-///   whitelisted)
+/// - tx is in allowlist, or
+/// - tx is executed by an approved governance proposal (no need to be allowed)
 pub fn is_valid_tx(ctx: &Ctx, tx_data: &Tx) -> VpResult {
-    if is_tx_whitelisted(ctx)? {
+    if is_tx_allowed(ctx)? {
         accept()
     } else {
         let proposal_id = tx_data
```

Comment:
I guess we dont need the `if` condition anymore as we are checking it in the protocol?

Reply:
Do we still need the entire function at all? If the tx comes from a transaction than we do check on the allowlist before vp execution and if it comes from governance it's allowed anyway. I think we need this only if there's another way to submit transactions to wasm which doesn't involve a normal transaction nor passing through governance?

Reply:
I removed it in https://github.com/anoma/namada/commit/dc50e580a19e3eb0180457899935237a0e1b2d6c